crates/srs/src/voice_stream.rs

```diff
@@ -163,26 +166,52 @@ async fn recv_updates(
     Ok(())
 }
 
-async fn send_updates(
+/**
+  Sends updates about the client to the server. If `game_source` is set,
+  the position and frequency from the latest received `GameMessage` is used.
+  Otherwise, the parameters set in the `client` struct are used.
+*/
```

Comment:
Would you mind changing those comments to doc comments using the three slash style? (`///` at the beginning of each line)